recipes/easylzma/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
```

Comment:
You can rename this file to test_package.c. As easyLZMA is a C project, you can test using a C code too.